lib/full-site-editing/template-loader.php

```diff
@@ -72,7 +72,7 @@ function gutenberg_override_query_template( $template, $type, array $templates =
 		$template_item_slug = gutenberg_strip_php_suffix( $template_item );
 
 		// Is this a custom template?
```

Comment:
Can we add a comment here explaining that we should remove this special case  once  it reaches Core and just consider wp_templates as valid templates before calling the hook?

Reply:
Done 👍